soc/realtek/ec/rts5912/reg/reg_uart.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2023 Realtek, SIBG-SD7
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _REALTEK_RTS5912_REG_UART_H
+#define _REALTEK_RTS5912_REG_UART_H
```

Comment:
```suggestion
#ifndef ZEPHYR_SOC_REALTEK_RTS5912_REG_UART_H
#define ZEPHYR_SOC_REALTEK_RTS5912_REG_UART_H
```

Reply:
We changed the identifiers in `reg_uart.h`, thanks.